aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -394,6 +394,15 @@ impl AptosVM {
         )?)
     }
 
+    fn get_senders(txn_data: &TransactionMetadata) -> Vec<AccountAddress> {
+        let mut res = vec![txn_data.sender];
+        res.extend(txn_data.secondary_signers());
+        if txn_data.sequence_number & GAS_PAYER_FLAG_BIT != 0 {
```

Comment:
Please add a comment why this pop is happening.

Reply:
+1 on leaving enough context behind so we dont accidentally break things in the future